compiler/damlc/daml-ide/src/DA/Daml/LanguageServer.hs

```diff
@@ -101,11 +105,11 @@ runLanguageServer
     :: ((FromServerMessage -> IO ()) -> VFSHandle -> ClientCapabilities -> IO IdeState)
     -> IO ()
 runLanguageServer getIdeState = do
-    let handlers = setHandlersKeepAlive <> setHandlersVirtualResource <> setHandlersCodeLens <> setIgnoreOptionalHandlers
+    let handlers = setHandlersKeepAlive <> setHandlersVirtualResource <> setHandlersCodeLens <> setIgnoreOptionalHandlers <> setCommandHandler
     LS.runLanguageServer options handlers getIdeState
 
 
 options :: LSP.Options
 options = def
     { LSP.codeLensProvider = Just $ CodeLensOptions $ Just False
-    }
+    }
```

Comment:
Why did you remove the trailing newline?

Reply:
:( Not sure adding it back